pymodbus/transport/transport.py

```diff
@@ -614,7 +612,7 @@ def write(self, data: bytes) -> None:
     # Dummy methods #
     # ------------- #
     def abort(self) -> None:
-        """Abort connection."""
+        """Old alias for closing the connection."""
```

Comment:
That is not a "old" ... it is needed because the parent class declares it virtual.